app/(app)/scan/scan-result.tsx

```diff
@@ -23,24 +47,34 @@ const getResultVariant = (result?: ScanResultEnum) => {
 // TODO - texts
 const ScanResultPage = () => {
   const { scanResult } = useLocalSearchParams<ScanResultSearchParams>()
+  const snackbar = useSnackbar()
+
+  if (!scanResult) {
+    snackbar.show('Nepodarilo sa načítať výsledok skenovania')
```

Comment:
I'm not sure what's our strategy for deling with these generic errors
I'd consider adding an error handler that would both show a toast and log an error somewhere
this is definitely outside of scope of this PR

Reply:
i have made changes in PR 27 (added error screen) so I will replace it after merging that PR